pkgs/development/python-modules/pytest-testinfra/default.nix

```diff
@@ -0,0 +1,48 @@
+{ lib
+, buildPythonPackage
+, fetchPypi
+, setuptools-scm
+, ansible-core
+, paramiko
+, pytest
+, pytest-cov
+, pytest-xdist
+, pywinrm
+, salt
+}:
+
+buildPythonPackage rec {
+  pname = "pytest-testinfra";
+  version = "8.1.0";
+
+  src = fetchPypi {
+    inherit pname version;
+    sha256 = "9b40828b58fb7ac2bff29cc1465934adf434f10dd5a108f60535dee52660a63d";
```

Comment:
Please use encoded hashes

Reply:
Thanks for your feedback. I tried to incorporate it but would rather not like to add another mean of contact.